toast-tk-runtime/src/main/java/io/toast/tk/runtime/report/XmlPrettierHelper.java

```diff
@@ -0,0 +1,104 @@
+package io.toast.tk.runtime.report;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class XmlPrettierHelper {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)